packages/main/src/pages/KnowledgeLibrary/Categories/Categories.jsx

```diff
@@ -0,0 +1,25 @@
+import navData from "../Navigation/NavData";
+import classes from "./Categories.module.css";
+import { Link } from "react-router-dom";
+
+function Categories(props) {
+  return (
+    <div className={classes["container"]}>
+      <p className={`${classes["header-p"]} ${props.className}`}>Categories</p>
+      {navData.map((item, i) => (
+        <div className={classes["item-box"]} key={i}>
```

Comment:
index is not a good key to map with

Reply:
done, i hope it's okay to use an obj as a key

Reply:
Give an example of what you'd like to try to use